modules/ocl/cl_demo_handler.cpp

```diff
@@ -79,7 +79,8 @@ CLDemoImageHandler::prepare_parameters (SmartPtr<VideoBuffer> &input, SmartPtr<V
     work_size.local[0] = 8;
     work_size.local[1] = 4;
 
-    _copy_kernel->set_arguments (args, work_size);
+    XCamReturn ret = _copy_kernel->set_arguments (args, work_size);
+    CHECK (ret, "copy_kernel set arguments failed.");
```

Comment:
CHECK is defined in test_common.h for unit tests code implementation, it will case build break. Please do not use here.